### ISSUE TITLE
[FEATURE] Build ML4W Apps during install.

### ISSUE THREAD

**Issue by 0blivi0nis** [mylinuxforwork/dotfiles#281]
Currently, the ML4W Apps are baked into the repository. This works out as a temporary solution but in the long run, does not make sense. Instead of shipping the app images with the dotfiles, I believe they should be installed directly from your source repositories. This would make it easier for you to push updates to them in the future if necessary.

**Comment by mylinuxforwork** [comment]
@0blivi0nis I totally agree with you. I have already started to think about to create dedicated AURs for each app. What do you think?

**Comment by mylinuxforwork** [comment]
@0blivi0nis The upcoming 2.9.6.2 will install the apps into the correct system paths of your Arch Linux. Launch planned for this weekend. This is the first step to prepare AURs.

**Comment by NMartin354** [comment]
Correct path is /usr/local/bin or ~/.local/bin

**Comment by mylinuxforwork** [comment]
I have created flatpak for all apps. They are shipped with the version 2.9.8
